runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -105,12 +105,12 @@ private Notifier(
 		storeIcon();
 	}
 
-	public void notify(String message)
+	public void notify(String message, boolean... shouldPrint)
```

Comment:
This is not how you do optional arguments....

Reply:
Also I really dislike how you decided to special case chat notifier

Reply:
It is a sort of workaround for the lack of optional parameters in Java. 

As for the special case what do you mean? Currently if I have chat notifier set up and also have "Enable game message notification" enabled all messages that trigger a notification  duplicated in chat.

Reply:
Alright fixed it.